### DISCUSSION TITLE
Badge issue

### DISCUSSION THREAD

**Discussion by shyamprakash123** [community/community#72998] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I got `Pair Extraordinaire` badge but it is not showing in the Achievements section, but if I visit https://github.com/shyamprakash123?tab=achievements it is visible. Can anyone help me?

**Comment by ktauchathuranga** [comment, marked as answer]
yes,
go there and make sure you uncheck the following hide option,
take a look at the top left corner.

![image](https://github.com/community/community/assets/79928743/e99a0e7d-53dc-4246-ada3-a13a5b142cda)
